cursos/kubernetes/00_solucions/01_solucion/01_solucion.md

```diff
@@ -0,0 +1,42 @@
+# Ejercicio Kubernetes 2
+
+[link](https://formacion.4eixos.com/k8s/actividades/2/correndo_a_nosa_primeira_aplicacin_en_kubernetes.html) al ejercicio.
+
+Cuando despliego el Deploy y el servicio hago un curl a la direccion ip del serivicio y no devuelve nada.
+
+
+Si hago un curl a localhost da connection refused
+
+
+Si hago un forward funciona perfectamente.
+
+
+# SOLUCION:
+
+Penso que tes ao reves o targetPort e o port. O targetPort sería o porto do pod donde está arrancado o servicio, e o porte o porto que vai a expoñer o servicio ao exterior, podes ver máis info aquí.
```

Comment:
@pablosanchezpaz, please correct this question:
- Missing initial problem
- The question is in Spanish but the answer is in Gallician, please use Gallician as the course is in this language.
- *Tes ao revés o `targetPort` e o `port`*
- *porte -> port*
- missing link: *podes ver máis info **aquí***